pkg/cmd/cli/cmd/newapp.go

```diff
@@ -90,6 +90,19 @@ application is created.`
 
   // Search for "ruby" in stored templates and print the output as an YAML
   $ %[1]s new-app --search --template=ruby --output=yaml`
+
+	newAppNoInput = `You must specify one or more images, image streams, templates or source code locations to create an application.
+
+Suggestions:
+Use '-L' or '--list' flag to list all local templates and image streams that can be used to create an application
```

Comment:
We *always* use the Oxford comma when we are listing things like nouns,
ideas, or related thoughts.

On Wed, Aug 26, 2015 at 12:51 PM, Fabiano Franz <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/newapp.go
> <https://github.com/openshift/origin/pull/4375#discussion_r37998852>:
>
> > @@ -90,6 +90,19 @@ application is created.`
> >
> >    // Search for "ruby" in stored templates and print the output as an YAML
> >    $ %[1]s new-app --search --template=ruby --output=yaml`
> > +
> > +	newAppNoInput = `You must specify one or more images, image streams, templates or source code locations to create an application.
> > +
> > +Suggestions:
> > +Use '-L' or '--list' flag to list all local templates and image streams that can be used to create an application
>
> Should we have an Oxford comma here?
>
> You must specify one or more images, image streams, *templates,* or
> source code locations (...)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4375/files#r37998852>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift